src/site/topic19.rst

```diff
@@ -245,6 +245,118 @@ Computational Complexity
 Fibonacci
 ---------
 
+* Consider the Fibonacci numbers
+* If you are not familiar with this sequence, see if you can figure out how it's created
+
+    :math:`1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89, 144, 233, 377, 610, 987, 1597, 2584, 4181, 6765, ...`
```

Comment:
Minor, I usually see `f(0)=0, f(1)=1` since it more closely follows the [golden spiral](https://en.wikipedia.org/wiki/Golden_spiral), at least as _r_ approaches 0. It also gives a more satisfying recursive function in my opinion.